packages/scene-composer/src/components/panels/scene-settings/FogSettingsEditor.tsx

```diff
@@ -94,54 +93,53 @@ export const FogSettingsEditor: React.FC = () => {
 
   return (
     <React.Fragment>
-      <ExpandableInfoSection
-        title={intl.formatMessage({ description: 'ExpandableInfoSection Title', defaultMessage: 'Fog Settings' })}
-        defaultExpanded={false}
-      >
-        <SpaceBetween size='s'>
-          <Checkbox
-            data-testid='enable-fog-checkbox'
-            checked={!!fogSettings}
-            onChange={(e) => onToggleFog(e.detail.checked)}
-          >
-            {intl.formatMessage({ defaultMessage: 'Enable Fog', description: 'checkbox label' })}
-          </Checkbox>
-          <ColorSelectorCombo
-            color={`#${internalColor?.toString(16).padStart(6, '0')}`}
-            onSelectColor={(pickedColor) => onColorChange(pickedColor)}
-            onUpdateCustomColors={(chosenCustomColors) =>
-              setTagColorsSceneProperty(KnownSceneProperty.TagCustomColors, chosenCustomColors)
-            }
-            customColors={tagStyleColors}
-            colorPickerLabel={intl.formatMessage({ defaultMessage: 'Color', description: 'Color' })}
-            customColorLabel={intl.formatMessage({ defaultMessage: 'Custom colors', description: 'Custom colors' })}
-          />
-          <FormField label={intl.formatMessage({ defaultMessage: 'Near Distance', description: 'Form Field label' })}>
-            <Input
-              data-testid='fog-near-input'
-              value={String(internalNearDistance)}
-              type='number'
-              onBlur={onInputBlur}
-              onChange={onNearChange}
-              onKeyDown={(e) => {
-                if (e.detail.key === 'Enter') onInputBlur();
-              }}
+      <SpaceBetween size='s'>
+        <Checkbox
+          data-testid='enable-fog-checkbox'
+          checked={!!fogSettings}
+          onChange={(e) => onToggleFog(e.detail.checked)}
+        >
+          {intl.formatMessage({ defaultMessage: 'Enable Fog', description: 'checkbox label' })}
+        </Checkbox>
+        {!!fogSettings && (
+          <>
+            <ColorSelectorCombo
+              color={`#${internalColor.toString(16).padStart(6, '0')}`}
+              onSelectColor={(pickedColor) => onColorChange(pickedColor)}
+              onUpdateCustomColors={(chosenCustomColors) =>
+                setTagColorsSceneProperty(KnownSceneProperty.TagCustomColors, chosenCustomColors)
+              }
+              customColors={tagStyleColors}
```

Comment:
is this intentional to update `TagCustomColors` when changing Fog color?

Reply:
We don't have fog or background color lists from UX yet so this was a stop gap for the PoC to get feedback.  But I think we can make new fog and background custom colors lists now and just have them initialized to the same values as our tag colors but save custom colors off separately from tags.  Then when we get colors from UX we can just change the initialization and not have to change each editor as well.

Reply:
the tag custom colors are a list of custom colors the user previously has selected. it is also initialized with an empty list

Reply:
You're right, I was confusing it with the ColorSelectorCombos initial list of colors of non-custom colors which is set in TagColors.ts. But I'm breaking them up to all use their own custom save list even if the ColorSelectorCombo is still initialized from TagColors for non-custom colors.

Reply:
added fogCustom and backgroundCustom save system separate from tagCustom